server/routes/category.js

```diff
@@ -0,0 +1,16 @@
+import express from 'express';
+import Categories from '../controllers/Categories';
+import middlewares from '../middlewares';
+
+const route = express.Router();
+const { authorizeUser, verifyToken, createCategoryValidation } = middlewares;
+
+route.post(
+  '/create',
+  verifyToken,
+  authorizeUser(4),
```

Comment:
Don't admins and super-admins have sessions that should be checked?

Reply:
they do. Thanks for pointing this out. I have added this.